src/pset/map/input.rs

```diff
@@ -351,6 +351,30 @@ impl PsbtSighashType {
 
 impl Input{
 
+    /// Obtains the [`EcdsaSigHashType`] for this input if one is specified. If no sighash type is
+    /// specified, returns [`EcdsaSigHashType::All`].
+    ///
+    /// # Errors
+    ///
+    /// If the `sighash_type` field is set to a non-standard ECDSA sighash value.
+    pub fn ecdsa_hash_ty(&self) -> Option<EcdsaSigHashType> {
```

Comment:
Just a nit, goes also for the method on `PsbtSighashType`, but it seems weird to try save 2 characters and shorten `type` to `ty`..

Reply:
Would like to keep the names from rust-bitcoin here.

Reply:
@stevenroose it's not to save characters, it's because `ty` is a keyword and therefore can't be used for variable names .. so we are trying to keep the naming consistent between variables and methods.